web/themes/custom/cloudy/templates/search/search-api-page.html.twig

```diff
@@ -0,0 +1,23 @@
+{{ form }}
+
+{% if search_title %}
+  <h2>{{ search_title }}</h2>
+{% endif %}
+
+{% set pathParts = path('<current>')|split('/') %}
+
+{% if no_of_results %}
+  <p>{{ no_of_results }}. If you are not seeing result you want, it may not be migrated yet. Try your search on <a href="https://www.portlandoregon.gov/?mode=search&search_action=SearchResults&top_category_tree_id=25777&filter_category_tree_id=25776&search_category_tree_id=25777&submit=Search&search_words={{ pathParts[pathParts|length - 1] }}">PortlandOregon.gov</a>.</p>
+{% endif %}
+
+{{ no_results_found }}
+
+{% if no_of_results == 0 %}
+It may not be migrated yet. Try your search on <a href="https://www.portlandoregon.gov/?mode=search&search_action=SearchResults&top_category_tree_id=25777&filter_category_tree_id=25776&search_category_tree_id=25777&search_words={{ pathParts[pathParts|length - 1] }}&submit=Search">PortlandOregon.gov</a>.
```

Comment:
Please change this line to "The information you are looking for may not be migrated yet. Try your search on PortlandOregon.gov." and wrap it in a paragraph element so that the spacing is a bit better.

Reply:
Updated. The text was copied from the story description.